### ISSUE TITLE
VGG16 Imagenet weight initialization

### ISSUE THREAD

**Issue by mees** [lichengunc/mask-faster-rcnn#5]
Hi,
which VGG16 Imagenet pretrained pytorch model can we use to intialize your VGG16 network? I tried converting the original VGG16 caffemodel to pytorch, but then there were some keys missing in the state dict.
Thanks in advance!

**Comment by mees** [comment]
Also I read that some some pretrained VGG16 models expect the input in RGB, others in BGR etc. how is it for your implementation? Which pretrained VGG16 model should we use for your code?
